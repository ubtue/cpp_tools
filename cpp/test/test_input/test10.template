{IF x == "Bob"}Bob{ELSE}ELSE BRANCH{ENDIF}
{IF x == "Alice"}Alice{ELSE}ELSE BRANCH{ENDIF}
{IF x == "Joe"}Joe{ELSE}ELSE BRANCH{ENDIF}
{x}