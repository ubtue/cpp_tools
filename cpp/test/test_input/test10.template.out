Bob
ELSE BRANCH
ELSE BRANCH
Bob